README.md

```diff
@@ -30,14 +30,50 @@ The MotorEase code and data has been permanently archived on Zenodo at: (https:/
 
 - This project uses a GloVe embedding for textual similarities. However, GloVe embedding files are large and difficult to host on GitHub. Therefore, we have created a sampleGlove.txt file within the docker container to act as a dummy GloVe model in place of a real one. This text file is formatted the exact same way as a Glove model is normally formatted.
 
-<ins>Running the Image: </ins>
+<ins> Build 1: Running the Image with Local Build: </ins>
 
-- In order to run the image on the container, run this command ``` docker run -it --rm -v $(pwd)/container_files:/container_files itsarunkv/motorease-arm /bin/bash ``` (be sure to use the correct "and" or "arm" image name in this command).
+- With these instructions you will be able to run the Docker Image locally and propagate your changes to the Docker Container automatically.
+
+- This build uses the: glove.6B.100d from ```https://github.com/stanfordnlp/GloVe?tab=readme-ov-file#download-pre-trained-word-vectors```so please download the glove embedding here if you would like to use the same model.
+  - Other models can be used and to use those models please change MotorEase.py in the Code directory with the model you would like to use.
+  - This model was chosen becasue it simulated the same results as the original Docker Image. 
+
+- To build the image run:
+ ```bash
+ docker build -t motorease Code
+ ```
+
+- To run a container of the image please run the command below. This command mounts the code directory which allows developers to have their changes propagated to docker container automatically. 
+```bash
+docker run -it --rm -v $(pwd)/Data:/data -v $(pwd)/Code:/code -v $(pwd)/Output:/output --env DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix motorease
+``` 
+
+- To enter a container of the image without running the command to start to program:
+```bash
+docker run -it --rm -v $(pwd)/Data:/data -v $(pwd)/Code:/code -v $(pwd)/Output:/output --env DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix motorease /bin/bash
+``` 
+
+- To clean up the image:
+ ```bash
+ docker rmi motorease
+ ```
+
+- Note 1: Depending on the Glove embedding used, you may need over 64 GB of ram
+  - Docker image can easily be over 10 GB
+  - When the codes failed, all the memory is still there, meaning, every time you run a 10 GB big model and fail, you have 10 GB allocated ram not released
+
+- Note 2: The compilation is slow even with a 8-core CPU. So, you may experience a long delay when it comes to both running and building the program.
+
+<ins> Build 2: Running the Image: </ins>
+
+- In order to run the image on the container, run this command ``` docker run -it --rm -v $(pwd)/container_files:/container_files itsarunkv/motorease-arm /bin/bash ``` (be sure to use the correct "amd" or "arm" image name in this command).
```

Comment:
The format for Build 2 instructions is a little bit inconsistent with both Build 1 and 3. Would you be able to separate the code lines to make it consistent?

Reply:
Yep! I added that the instructions are for WSL and added your note on how to use it for Powershell:
![image](https://github.com/hemkan/MotorEase/assets/93550993/9c220fe2-b26c-4310-9afe-72cc4460fd15)